content/it/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,126 @@
+---
+title: I componenti di Kubernetes
+content_template: templates/concept
+weight: 20
+card: 
+  name: concepts
+  weight: 20
+---
+
+{{% capture overview %}}
+Facendo il deployment di Kubernetes, ottieni un cluster.
+{{< glossary_definition term_id="cluster" length="all" prepend="Un cluster è">}}
+
+Questo documento describe i diversi componenti che sono necessari per avere 
+un cluster Kubernetes completo e funzionante.
+
+Questo è un diagramma di un cluster Kubernetes con tutti i componenti e le loro relazioni.
+
+![Components of Kubernetes](/images/docs/components-of-kubernetes.png)
+
+{{% /capture %}}
+
+{{% capture body %}}
+## Componenti del Master
+
+I componenti del Master forniscono la control plane del cluster.
+I componenti del Master fanno decisioni globali sul cluster (ad esempio, lo scheduling), e individuano e rispondono a eventi derivanti dal cluster (ad esempio, l'avvio di un nuovo {{< glossary_tooltip text="pod" term_id="pod">}} quando il valore `replicas` di un deployment non è soddisfatto).
+
+I componenti del Master possono essere eseguiti su qualsiasi nodo del cluster.
+Solitamente, per motivi di facilità, gli script di installazione tendono a eseguire tutti i componenti del Master sulla stessa macchina, senza l'utilizzo di container.
+Vedi [creare un cluster in High-Availability](/docs/admin/high-availability/) per un esempio di un'installazione multi-master.
+
+### kube-apiserver
+
+{{< glossary_definition term_id="kube-apiserver" length="all" >}}
+
+### etcd
+
+{{< glossary_definition term_id="etcd" length="all" >}}
+
+### kube-scheduler
+
+{{< glossary_definition term_id="kube-scheduler" length="all" >}}
+
+### kube-controller-manager
+
+{{< glossary_definition term_id="kube-controller-manager" length="all" >}}
+
+Questi controller includono:
+
+  * Node Controller: Responsabile dell'identificazione e della risposta ad eventi in cui i nodi diventano non disponibili.
```

Comment:
dell'identificazione e della risposta ad eventi in cui i nodi diventano non disponibili --> del monitoraggio dei nodi del cluster, e.g. della gestione delle azioni da eseguire quando un nodo diventa non disponibile